src/Profile.cpp

```diff
@@ -1198,6 +1204,11 @@ void Profile::LoadSongsFromDir(RString const& dir, ProfileSlot prof_slot)
 
 		StripCvsAndSvn(song_folders);
 		StripMacResourceForks(song_folders);
+		RString groupName = RageUtil::GetCustomSongPath();

```

Comment:
Seems like this groupName isn't used until later. Let's move it closer to it's actual usage below.